src/transport/CASESession.h

```diff
@@ -126,6 +158,30 @@ class DLL_EXPORT CASESession
 
     Transport::PeerConnectionState & PeerConnection() { return mConnectionState; }
 
+    /** @brief Serialize the Pairing Session to a string.
```

Comment:
Does not seem to serialize to a 'string' but rather to serialized output.

Could we drop non-obvious doxygens? Return error on error and no-error on success is obvious and so is the fact that 'Serialize serializes things'.